integration

```diff
@@ -472,6 +472,7 @@
   "jmcruvellier/little_monkey",
   "jnxxx/homeassistant-dabblerdk_powermeterreader",
   "jobvk/Home-Assistant-Windcentrale",
+  "joemcc-90/leeds-bins-hass",
```

Comment:
FYI: <https://github.com/joemcc-90/leeds-bins-hass/blob/main/hacs.json#L6>
`persistent_directory` will be removed, as it does not work if the user is not using HACS to update.